### DISCUSSION TITLE
[11 Settembre 2023] - Cache

### DISCUSSION THREAD

**Discussion by notedo** [CS-Swap/Architettura-degli-Elaboratori#44] [Cache]
### Traccia Esercizio

![278167251-00020545-af0c-4717-bd91-fda33c517ef7](https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/110790224/a5cc8872-8e92-49ea-8a64-c21b386603f7)

**Comment by erinrrr** [comment]
![image](https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/147604739/ba36c2fc-5c5c-49ad-8d43-b6a3372ad5bb)
![image](https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/147604739/23479976-ce69-40a1-9808-78824b4ec302)
![image](https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/147604739/453729ca-a1c6-4ed0-b06a-d781d9e7062f)
![image](https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/147604739/c079a621-7c6b-4c28-9601-bdb312284d82)

**Comment by Divine-Sunderer** [comment]
<img width="2935" alt="11 Settembre 2023  - Cache" src="https://github.com/CS-Swap/Architettura-degli-Elaboratori/assets/157913185/b9d9589b-b7c7-498f-be9f-b12b351e1557">
